src/IceRpc.Protobuf/ProtobufEncodeOptions.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) ZeroC, Inc.
+
+using System.Buffers;
+using System.IO.Pipelines;
+
+namespace IceRpc.Protobuf;
+
+/// <summary>Represents a property bag used to configure the encoding of payloads.</summary>
+public sealed class ProtobufEncodeOptions
+{
+    /// <summary>Gets the default instance of <see cref="ProtobufEncodeOptions" />.</summary>
+    public static ProtobufEncodeOptions Default { get; } = new();
+
+    /// <summary>Gets the pipe options that the Slice engine uses when creating pipes. The Slice engine creates a pipe
```

Comment:
No "Slice engine".

It's really the IceRPC + Protobuf integration or IceRPC + Protobuf for short. (Same for Slice btw).